src/nr_openai_observability/build_events.py

```diff
@@ -193,19 +266,35 @@ def build_embedding_event(response, request, response_headers, response_time):
 
     embedding = {
         "id": embedding_id,
+        "request_id": response_headers.get("x-request-id", ""),
         "input": request.get("input")[:4095],
         "api_key_last_four_digits": f"sk-{response.api_key[-4:]}",
         "timestamp": datetime.now(),
-        "response_time": int(response_time * 1000),
         "request.model": request.get("model") or request.get("engine"),
         "response.model": response.model,
-        "usage.total_tokens": response.usage.total_tokens,
-        "usage.prompt_tokens": response.usage.prompt_tokens,
-        "api_type": response.api_type,
         "vendor": "openAI",
         "ingest_source": "PythonSDK",
-        "organization": response.organization,
-        "api_version": response_headers.get("openai-version"),
+        **compat_fields(["response_time", "duration"], int(response_time * 1000)),
+        **compat_fields(
+            ["usage.total_tokens", "response.usage.total_tokens"],
+            response.usage.total_tokens,
+        ),
+        **compat_fields(
+            ["usage.prompt_tokens", "response.usage.prompt_tokens"],
+            response.usage.prompt_tokens,
+        ),
+        **compat_fields(
+            ["response.api_version", "response.headers.llmVersion"],
+            response_headers.get("openai-version"),
+        ),
+        **compat_fields(["api_type", "response.api_type"], response.api_type),
+        **compat_fields(
+            ["api_version", "response.api_version"],
```

Comment:
Do we need a second copy of `response_headers.get("openai-version")` under new names?

Reply:
Nope. Good call. Looks like we ended up with a duplicate